### PR TITLE
remove some tests warnings

### PR DESCRIPTION
**Pull request by edgarriba**
#### Changes

removes some warnings during testing

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: test/geometry/test_line.py [hunk 1]
```diff
@@ -3,33 +3,40 @@
 from torch.autograd import gradcheck
 
 from kornia.geometry.line import ParametrizedLine, fit_line
-from kornia.testing import assert_close
 
 
-class TestParametrizedLine:
+class BaseTester:
+    def assert_close(self, actual, expected, rtol=None, atol=None):
```
Comment (edgarriba) [inline test/geometry/test_line.py:9]: @MrShevan this is another common pattern i found for testing

Comment (MrShevan) [reply test/geometry/test_line.py:9]: May we merge this [PR](https://github.com/kornia/kornia/pull/1809) into master and merge new master into this branch? 
Then we can add this logic to method `assert_close` from existing `BaseTester` from `kornia.testing`?
```
if isinstance(actual, torch.nn.Parameter):
    actual = actual.data
if isinstance(expected, torch.nn.Parameter):
    expected = expected.data
```

Comment (edgarriba) [reply test/geometry/test_line.py:9]: defenitely

**Comment by MrShevan** [comment]
Hi, @edgarriba!) I think we can close this PR, it is no longer Relevant.

**Pull request closed without merging**